lib/queuePopulator/LogReader.js

```diff
@@ -355,7 +355,30 @@ class LogReader {
         });
     }
 
+    _isSupportedOperation(record, entry) {
+        // List of methods defined in Metadata/lib/protocol.json.
+        if (record.method && record.method < 10) {
+            if (!entry.type ||
+                entry.type === 'put' ||
+                entry.type === 'del' ||
+                entry.type === 'delete') {
+                return true;
+            }
```

Comment:
I don't think we need this block, as it would be isofunctional without it for all methods < 10 (and we save a bit of processing time as well).

Ideally I'd say it's up to the extension to check what it should consider and what to ignore, but I am aligned that this workaround is the least risky and intrusive considering the existing code.

Reply:
Removed the checks on `type`.

I also changed the condition to not filter out operations without a `method` (for merging in in `8` and `9` branches, not sure `method` exists in Artesca).